--- src/components/ui/sidebar.tsx
+++ src/components/ui/sidebar.tsx
@@ -10,7 +10,6 @@
 import { Input } from "@/components/ui/input";
 import { Separator } from "@/components/ui/separator";
 import { Sheet, SheetContent } from "@/components/ui/sheet";
-import { isTimerRunning, timerStartTimestamp } from "@/lib/context";
 import { Skeleton } from "@/components/ui/skeleton";
 import {
     Tooltip,
@@ -23,6 +22,8 @@
 import { cn } from "@/lib/utils";
+import { isTimerRunning, timerStartTimestamp } from "@/lib/context";
+import { toast } from "react-toastify";

 const SIDEBAR_COOKIE_NAME = "sidebar:state";
 const SIDEBAR_COOKIE_MAX_AGE = 60 * 60 * 24 * 7;
@@ -355,7 +356,12 @@
             )}
             onClick={(event) => {
                 if ($isTimerRunning || $timerStartTimestamp) {
-                    alert("You cannot open the right sidebar after the game has started.");
+                    toast.error(
+                        "Cannot configure Game State after the game has started.",
+                        {
+                            toastId: "configure-game-state-error",
+                        }
+                    );
                 } else {
                     onClick?.(event);
                     toggleSidebar();
